src/Symfony/Component/Yaml/Tests/ParserTest.php

```diff
@@ -66,6 +66,22 @@ public function getDataFormSpecifications()
         return $tests;
     }
 
+    public function testLineBreaksBeforeDocumentSeparatorAndComment()
```

Comment:
the test name does not describe what it tests, as it actually tests having a comment before the document separator, not having a line break before the comment